safegraph_py_functions/safegraph_py_functions.py

```diff
@@ -143,7 +151,11 @@ def unpack_json_and_merge(df, json_column='visitor_home_cbgs', key_col_name='vis
     df = df.merge(df_exp, left_index=True, right_index=True).reset_index(drop=True)
     return df
 
-def explode_json_array(df_, array_column = 'visits_by_day', value_col_name='day_visit_counts',place_key='safegraph_place_id', file_key='date_range_start', array_sequence='day', keep_index=False, verbose=True, zero_index=False):
+def explode_json_array(df_, array_column = 'visits_by_day', value_col_name=None, place_key='safegraph_place_id', file_key='date_range_start', array_sequence=None, keep_index=False, verbose=True, zero_index=False):
+    if (array_sequence is None):
+      array_sequence = array_column + 'sequence'
```

Comment:
@Trippl7777  can we make this `_sequence` instead of `sequence` for consistency?